data/locale/en-US.ini

```diff
@@ -109,9 +111,9 @@ AdvSceneSwitcher.macroTab.onChange="Perform actions only on condition change"
 AdvSceneSwitcher.macroTab.defaultname="Macro %1"
 AdvSceneSwitcher.macroTab.defaultGroupName="Group %1"
 AdvSceneSwitcher.macroTab.exists="Macro name exists already"
-AdvSceneSwitcher.macroTab.groupDeleteConfirm="Are you sure you want to delete \"%1\" and all its elements?"
```

Comment:
I think you accidentally one of the translation keys in the other language's translation files.
`AdvSceneSwitcher.macroTab.groupDeleteConfirm` seems to also exist in `de-DE.ini`.
And `AdvSceneSwitcher.macroTab.deleteSingleMacroPopup.text` in `zh-CN.ini` was probably intended to be `AdvSceneSwitcher.macroTab.removeSingleMacroPopup.text`.

Just FYI - you can spot those points in the `Check locale` step of the CI scripts:
https://github.com/WarmUpTill/SceneSwitcher/actions/runs/8211617652/job/22460785463

Reply:
That's a regression since it was reported automatically before I'm sure. Let me see if I can fix it with those BTW

Reply:
Wasn't regression, it just partially reported some issues as warnings I guess. I changed it to errors so that mistakes are caught more easily. And I removed checks of comments. So that you can leave some stuff like the sequence line commented and the translators can reuse it for new format of string later, which I guess was partially a reason for these warnings.